lib/system/generic/xlnx/CMakeLists.txt

```diff
@@ -8,4 +8,10 @@ collect (PROJECT_LIB_HEADERS sys.h)
 
 collect (PROJECT_LIB_SOURCES irq.c)
 
+include(CheckSymbolExists)
+check_symbol_exists(SDT "bspconfig.h" HAS_SDT)
+if (HAS_SDT)
+  collect (PROJECT_LIB_HEADERS sdt.h)
+endif()
+
```

Comment:
could you remove the remaining  `# vim ...` line above ( can be done in this commit)

Reply:
will do